pkg/hydrate/controller_test.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package hydrate
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"kpt.dev/configsync/pkg/api/configsync/v1beta1"
+	"kpt.dev/configsync/pkg/importer/filesystem/cmpath"
+	ft "kpt.dev/configsync/pkg/importer/filesystem/filesystemtest"
+)
+
+var commitHash = "1234567890abcdef"
+var differentHash = "abcdef1234567890"
+
+func TestRunHydrate(t *testing.T) {
+	testCases := []struct {
+		name         string
+		wantedRes    bool
```

Comment:
Suggest renaming `wantedRes` to `commitChanged`.

Reply:
Yeah